app/src/main/java/com/example/cocktailme/RecipeDetailsActivity.java

```diff
@@ -0,0 +1,31 @@
+package com.example.cocktailme;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+import android.widget.TextView;
+
+import com.example.cocktailme.models.Ingredients;
+
+import org.parceler.Parcels;
+
+public class RecipeDetailsActivity extends AppCompatActivity {
+
+    Ingredients ingredient;
+    TextView recTitle;
+    TextView recDesc;
```

Comment:
NIT, you can declare multiple variables of the same type in one line of code, so you can instead do this
```suggestion
    TextView recTitle, recDesc;
```

Reply:
i didnt know that! thanks for tip :)